app/modules/job_control/models.py

```diff
@@ -12,7 +12,7 @@
 class JobControl(db.Model, HoustonModel):
     guid = db.Column(db.GUID, default=uuid.uuid4, primary_key=True)
     asset_group_sighting_uuid = db.Column(db.GUID, nullable=True)
-    annotation_uuid = db.Column(db.GUID, nullable=True)
+    sighting_uuid = db.Column(db.GUID, nullable=True)
```

Comment:
Don't worry about the changes in this file, the whole file is stripped out in PR 217